examples/text_classification/raspberry_pi/README.md

```diff
@@ -0,0 +1,47 @@
+
+# MediaPipe Text Classification with Raspberry Pi
```

Comment:
@PaulTR 
I don't think we need text samples because it doesn't add any extra value from the Colab notebook. WDYT?